### ISSUE TITLE
changes in encoing setting

### ISSUE THREAD

**Issue by Mister-XY** [RandomEngy/VidCoder#489]
Hello, every time I change something in the encoding settings and load a different profile, I'm asked if I want to save the new setting. But I do not want this and then the window annoys over time. Is it possible to disable this window?
Is it possible to set a tick below Global Settings if I want to be asked for it or not.
![option](https://user-images.githubusercontent.com/10309056/51804328-a5c68880-225f-11e9-9046-3df92a896cf8.png)

**Comment by Mister-XY** [comment]
The next problem is under custom.
If i go to custom, i have a old custom setting. Is it not bette if i have the setting from automatic, what i can change?
![normal](https://user-images.githubusercontent.com/10309056/51806115-18daf980-2276-11e9-9c01-c057ccc5ee2c.png)
![user](https://user-images.githubusercontent.com/10309056/51806117-1d9fad80-2276-11e9-901d-8d0702e9447b.png)

**Comment by RandomEngy** [comment]
Do you want to automatically save all changes then?

**Comment by Mister-XY** [comment]
No, i dont want save anything. A want to let the preset how there are. If i want save anything, i can go to save.

**Comment by RandomEngy** [comment]
I don't think setting it up to throw away changes by default would be a very widely used setting. It goes against the behavior of basically every editor.
